apps/jsonrpc2/lib/bridge/sync.ex

```diff
@@ -100,4 +100,32 @@ defmodule JSONRPC2.Bridge.Sync do
         nil
     end
   end
+
+  def get_uncle_count_by_block_hash(hash) do
+    state_trie = get_last_sync_state().trie
+
+    case Block.get_block(hash, state_trie) do
+      {:ok, block} ->
+        block.ommers
+        |> Enum.count()
+        |> Exth.encode_unsigned_hex()
+
+      _ ->
+        nil
+    end
+  end
+
+  def get_uncle_count_by_block_number(number) do
```

Comment:
Seeing this more, we should replace `Block.get_block_by_number` with 
`Block.get_block(id, trie) when is_integer(id)` and `Block.get_block(id, trie) when is_binary(id)` overloads, so that we don't have to write two functions for all callers of `get_block` / `get_block_by_number`. This is really my fault, as I created `get_block_by_number`.